js/main.js

```diff
@@ -0,0 +1,67 @@
+/*
+  Author: David Lin
+  Group: Team Stimulus
+  Created At: 2/28/15
+
+  This is the core JS file that contains all of the bindings for DOM
+  actions such as file browsing and submitting the form
+
+  All code in this file was authored by David Lin
+*/
+
+$(function() {
+  var directory = new air.File();
+  var picturePath = "";
+  var ratingsFile;
+  var pictures = [];
+
+  /*
+    When the #picture-directory input is clicked, use AIR's browseForDirectory() method
+    When the directory is selected, save the path and display it in the form.
+  */
+  $('#picture-directory').click(function(e) {
+    e.preventDefault();
+    directory.addEventListener(air.Event.SELECT, function(e) {
+      picturePath = directory.nativePath;
+      $('#picture-directory').text(picturePath);
+    });
+    directory.browseForDirectory("Choose the picture directory");
+  });
+
+  /*
+    When submit is clicked on the form, parse all of the input into variables
+    and use the Parse module to parse the ratings file.
+  */
+  $('#input-form').submit(function(e) {
```

Comment:
I'd actually like to see this function cut down a bit, or have some inline comments.

both for the code style guide and for future readability